README.md

```diff
@@ -1,79 +1,127 @@
 # ProjectAutomation
+
```

Comment:
Adding whitespace like this in a README.md doesn't do anything.

Reply:
Markdown plugin for vscode did that. My bad.

Reply:
No problem at all, thanks for submitting the pull request!